docs/workflow-diagram/workflow.mmd

```diff
@@ -0,0 +1,57 @@
+graph LR
+
+  subgraph "ClinVar"
+    CLINVAR_TSV[Summary <br> TSV]
+    CLINVAR_VCF[Summary <br> VCF]
+    CLINVAR_XML[Full <br> XML]
+  end
+
+  subgraph "Manual curation protocol"
+    CLINVAR_TSV
+    --> TRAIT_MAPPING_PIPELINE([Trait <br> mapping <br> pipeline])
+    --> AUTOMATED_MAPPINGS[Automated <br> mappings] & MAPPINGS_REQUIRING_CURATION[Mappings <br> requiring <br> curation]
+
+    MAPPINGS_REQUIRING_CURATION
+    --> MANUAL_CURATION([Manual <br> curation])
+    --> MANUALLY_CURATED_MAPPINGS[Manually curated <br> mappings]
+
+    AUTOMATED_MAPPINGS & MANUALLY_CURATED_MAPPINGS
+    --> FINISHED_MAPPINGS[Finished <br> mappings]
+
+    MANUALLY_CURATED_MAPPINGS
+    --> SUBMIT_FEEDBACK_TO_EFO([Submit feedback <br> to EFO])
+  end
+
+  subgraph "Evidence string generation protocol"
+    CLINVAR_XML
+    --> CONVERT_CLINVAR([Convert <br> and filter <br> ClinVar XML])
+    --> CLINVAR_JSON[ClinVar <br> filtered JSON]
+
+    CLINVAR_VCF
+    --> PREDICT_CONSEQUENCES_VEP([Predict functional <br> consequences <br> using VEP])
+
+    CLINVAR_TSV
+    --> PREDICT_CONSEQUENCES_REPEATS([Predict functional <br> consequences <br> for repeats])
+
+    PREDICT_CONSEQUENCES_VEP & PREDICT_CONSEQUENCES_REPEATS
+    --> CONSEQUENCES[Functional <br> consequence <br> predictions]
+
+    CLINVAR_JSON & CONSEQUENCES & FINISHED_MAPPINGS
+    --> EVIDENCE_STRING_GENERATION([Evidence string <br> generation pipeline])
+  end
+
+  subgraph End result
+    EVIDENCE_STRING_GENERATION
+    --> EVIDENCE_STRINGS[Evidence <br> strings] & ZOOMA_FEEDBACK[ZOOMA <br> feedback]
+  end
+
+classDef pipeline fill:#0f0
+
+class TRAIT_MAPPING_PIPELINE pipeline
+class SUBMIT_FEEDBACK_TO_EFO pipeline
+class MANUAL_CURATION pipeline
+
+class CONVERT_CLINVAR pipeline
+class PREDICT_CONSEQUENCES_REPEATS pipeline
+class PREDICT_CONSEQUENCES_VEP pipeline
+class EVIDENCE_STRING_GENERATION pipeline
```

Comment:
Awesome stuff!!

Reply:
Yeah, Mermaid is cool! The design of the diagrams it produces is not exactly award winning, but it's so much better than drawing it by hand. I'm sick and tired of moving boxes & arrows around in Inkscape or Google Slides